### PR TITLE
Playlist feature

### PR DESCRIPTION
**Pull request by cjinc121**
- added playlist create delete feature 
- playlists page
- video add delete to playlist

### CODE REVIEW

File: BingeIT/src/App.js [hunk 1]
```diff
@@ -6,6 +6,11 @@ import { Header } from "./components/Header/Header";
 import { Sidebar } from "./components/Sidebar/Sidebar";
 import { Footer } from "./components/Footer/Footer";
 import { VideoPage } from "./pages/VideoPage/VideoPage";
+import { LoginPage } from "./pages/AuthPage/Loginpage";
+import { Profile } from "./pages/AuthPage/Profile";
+import { PrivateRoute } from "./routes/PrivateRoute";
+import { Playlists } from "./pages/PlaylistsPage/PlaylistsPage";
+import PlaylistPage from "./pages/PlaylistPage/PlaylistPage";
```
Comment (rahulcs754) [inline BingeIT/src/App.js:13]: You can create index file and export there all file so you have to only here import using one line

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good 
just small suggestion

File: BingeIT/src/components/Videoinfo/horizontal/VideoHorizontal.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import "./VideoHorizontal.css";
+import { BsThreeDotsVertical } from "react-icons/bs";
+import { CgPlayListRemove } from "react-icons/cg";
+import { useNavigate, useParams } from "react-router-dom";
+import { useAuth } from "../../../context/auth-context";
+import { removeVideoFromPlaylistService } from "../../../services/playlistCalls";
+function VideoHorizontal({ video }) {
+  const { playlistId } = useParams();
+  console.log(playlistId);
```
Comment (SukanyaSen530) [inline BingeIT/src/components/Videoinfo/horizontal/VideoHorizontal.js]: Remove the console.log

**Review by SukanyaSen530 — APPROVED** [review]

File: BingeIT/src/App.js (hunk 1 above)
Comment (cjinc121) [reply BingeIT/src/App.js:13]: okay

File: BingeIT/src/components/Videoinfo/horizontal/VideoHorizontal.js (hunk 2 above)
Comment (cjinc121) [reply BingeIT/src/components/Videoinfo/horizontal/VideoHorizontal.js]: okay,done

**Pull request merged**